doc/man1/openssl-cmp.pod.in

```diff
@@ -334,7 +334,7 @@ is provided via the B<-newkey> or B<-key> options.
 
 =item B<-csr> I<filename>
 
-CSR in PKCS#10 format to use in legacy P10CR messages.
+PKCS#10 CSR in PEM oder DER format to use in legacy P10CR messages.
```

Comment:
We use English instead of German. :laughing:

Reply:
> We use English instead of German. laughing

I didn't get your point - can you explain in German instead? :wink:

I changed this because the OpenSSL apps tend to use the work 'format' more for the encoding format such as text/PEM or binary/ASN.1/DER, and PKCS#10 (as opposed to CMP or CMS) is the type of the CSR at a different level.
Does the following sound better to you?
```
CSR in PKCS#10 style (given in PEM or DER format) to use in legacy P10CR messages.

Reply:
s/PEM oder DER/PEM or DER/

Reply:
I explained in sed language.

Reply:
> s/PEM oder DER/PEM or DER/

Ah yes, I must have been blind (or: I must have had tomatoes on my eyes)!
Fixed.